### DISCUSSION TITLE
DietPi-Software | Stacer

### DISCUSSION THREAD

**Discussion by jalsco** [MichaIng/DietPi#5725] [Software Request] [answered]
I would like to run  Stacer on my DietPi  

In fact - I did install it manually -  by download the DEB file and install it. But, even install is successful and the Stacer icon shows up in the LXDE desktop under "Accessories" - but then - it just does not work when try to run it... Click on the icon and it does nothing... 

I also run some Rasberry Pi computers - and Stacer does work on those. 

Here are the links: 

https://www.addictivetips.com/ubuntu-linux-tips/install-and-use-stacer-on-linux/
https://github.com/oguzhaninan/Stacer/releases

**Comment by Joulinar** [comment, marked as answer]
You are running an ARM based SBC? If yes, the Stacer Debian package will not work, because it is a x64 (AMD64) package only.

Anyway, it's available from Debian global package repository and can be installed easily 
https://packages.debian.org/search?keywords=stacer&searchon=names&suite=stable&section=all

```
apt purge stacer # purge current version
apt install stacer # install from Debian repository
```

**Comment by jalsco** [comment]
Yes - am running arm64 system. 

Thanks for your advice !!   I followed those two steps that you gave - and presto 😊 - it is now running on my system. Thank you so much ! 🙏🏻
